wercker.yml

```diff
@@ -54,6 +54,18 @@ build:
           echo "build is skipped ..."
         fi
 
+  - script:
+      name: mvn clean verify
+      code: |
+        if [[ $SKIP_CI == 'false' ]]; then
+          cd sevntu-checks
+          echo "Command: mvn clean verify"
+          mvn clean verify
```

Comment:
What does this do that https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/5ce28b1218e681ae1a4796b651e125c0bdd615c9/wercker.yml#L51 doesn't?

Reply:
` mvn -e verify -Pno-validations` only builds the jar and runs no unit tests or any other validations.

Reply:
Relook at the new URL. I pasted the wrong thing.

Reply:
Also, travis appears to be disabled.

Reply:
@romani Is something wrong with travis? I am not seeing any "disabled" in it but it's not running atleast in this PR.

Reply:
> Relook at the new URL

I did, the same still holds. No unit tests are ran.

Reply:
Also, from travis-ci.org:
`Since June 15th, 2021, the building on travis-ci.org is ceased. Please use travis-ci.com from now on.`

Reply:
Travis is not running on any PRs in any repo.

Reply:
Ok, without a working travis I can't show if I am just missing the command. I find it extremely odd we have not had something running tests all this time.

Reply:
> I find it extremely odd we have not had something running tests all this time.

After scanning through `travis.sh` and `travis.yml`, this appears to be the case.

Reply:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/9deadf04e0afa03ef4e94d1ccfb4576234e16ca2/.ci/travis.sh#L56

this suppose to do verify, as verify is part of install phase.

https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/9deadf04e0afa03ef4e94d1ccfb4576234e16ca2/sevntu-checks/pom.xml#L790

Reply:
I can confirm that `mvn -e -Pcoverall install ` does run unit tests. Sorry about the confusion.